tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,48 +87,69 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val wskpropsBackup = TestUtils.backupWskProps()
```

Comment:
A while ago, Carlos and I had a discussion about the use of WSK_CONFIG_FILE.  The basic issue boiled down to opening the door to the possibility of a customer saving .wskprops files under source control (i.e. staging configuration, production configuration, etc) coupled with the possible future addition of the secure/insecure setting being added to this properties file.  We wanted to make sure deployments did not result in the CLI accidentally being configured for insecure operation.  So, the thought was that this env variable would eventually go away; so I was making preparations for that day.